database/schema.sql

```diff
@@ -3,29 +3,33 @@ CREATE DATABASE sdcphotogallery;
 
 \c sdcphotogallery;
 
-CREATE TABLE users(
-id BIGSERIAL NOT NULL PRIMARY KEY,
-email varchar(30) NOT NULL,
+CREATE TABLE IF NOT EXISTS users(
+id SERIAL NOT NULL PRIMARY KEY,
+email varchar(50) NOT NULL,
 first_name varchar(20) NOT NULL,
-last_name varchar(20) NOT NULL,
+last_name varchar(20) NOT NULL
 );
 
-CREATE TABLE rooms (
-id BIGSERIAL NOT NULL PRIMARY KEY,
-user_Id BIGINT REFERENCES users(id), --not sure if necessary (owner owns rooms?)
-title varchar (30) NOT NULL,
-address varchar (70) NOT NULL,
+CREATE TABLE IF NOT EXISTS rooms (
+id SERIAL NOT NULL PRIMARY KEY,
+title varchar (80) NOT NULL,
+address varchar (70) NOT NULL
 );
 
-CREATE TABLE list (
-id BIGSERIAL NOT NULL PRIMARY KEY,
-user_Id BIGINT REFERENCES users(id),
-name varchar(30),
-room_id REFERENCES rooms(id)
+CREATE TABLE IF NOT EXISTS list (
+id SERIAL NOT NULL PRIMARY KEY,
+user_id INTEGER REFERENCES users(id),
+name varchar(80),
+room_id INTEGER REFERENCES rooms(id)
 );
 
-CREATE TABLE room_photos (
-id BIGSERIAL NOT NULL PRIMARY KEY,
+CREATE index user_id_room_id_list_index ON list (user_id, room_id);
```

Comment:
You would probably want to create all of your indexes at the very end of your script after all data is inserted into DB --> no need to worry about foreign keys reference from other tables